dbsnp-importer/src/main/resources/application.properties

```diff
@@ -7,8 +7,8 @@ spring.jpa.generate-ddl=false
 
 ## Job repository database
 # See JobRepositoryConfiguration, use maven profiles to put the next properties in the environment:
-spring.datasource.driver-class-name=@eva.jobrepository.driver-class-name@
-spring.datasource.url=@eva.jobrepository.url@
-spring.datasource.username=@eva.jobrepository.username@
-spring.datasource.password=@eva.jobrepository.password@
+#spring.datasource.driver-class-name=@eva.jobrepository.driver-class-name@
```

Comment:
why have been those properties commented? to force adding them to the external application.properties? The job repository is not going to change between imports, so I think is good to have them "in the jar"

Reply:
I didn't intend to push these changes, I will revert them...